tools/invoke-tests

```diff
@@ -77,8 +77,8 @@ args=()
 # use unbuffer if it was found so e.g. timestamps on OBS builds will be useful
 # note: When executing tests with prove it seems that the output is only flushed after one test has finished unless there's a terminal.
 [[ $unbuffer_path ]] && [[ $unbuffer_path != 'UNBUFFER_PATH-NOTFOUND' ]] && args+=("$unbuffer_path")
-# split TESTS on white-spaces to allow specifying multiple tests as documented
-TESTS=${TESTS:-"-r t xt"}
+# run tests from t only by default
+TESTS=${TESTS:-"t"}
```

Comment:
I don't understand this part. When you just specify "t" that means we now exclude "xt" tests?

Reply:
Yes. From the default which test-perl-testsuite relies on.

I guess that doesn't need to change, though. I'm passing TESTs= instead.

Reply:
Still the problem is that now xt is not executed anymore, see https://github.com/os-autoinst/os-autoinst/actions/runs/9679787998/job/26706502005?pr=2507

Reply:
> Still the problem is that now xt is not executed anymore, see https://github.com/os-autoinst/os-autoinst/actions/runs/9679787998/job/26706502005?pr=2507

That's what we want. the xt folder is covered in the author tests.

Reply:
ah, right. But I would very much prefer if the default for invoke-tests is to run all the tests. Why should invoke-tests just run for "t"?

Reply:
The default is to run `-r t xt` now, and this PR doesn't change this (anymore).

Reply:
true, not included anymore